app/controllers/registrations_controller.rb

```diff
@@ -62,8 +61,22 @@ def update_first_user_permissions(resource)
   end
 
   def recaptcha_verified?
-    recaptcha_params = { secret_key: Settings::Authentication.recaptcha_secret_key }
-    params["g-recaptcha-response"] && verify_recaptcha(recaptcha_params)
+    if ReCaptcha::CheckRegistrationEnabled.call
+      recaptcha_params = { secret_key: Settings::Authentication.recaptcha_secret_key }
+      params["g-recaptcha-response"] && verify_recaptcha(recaptcha_params)
+    else
+      true
+    end
+  end
+
+  def resolve_profile_field_issues
+    # Run this data update script when in a state of "first user" in the event
+    # that we are in a state where this was not already run.
+    # This is likely only temporarily needed.
```

Comment:
Is this still needed? Since we're here cleaning up already I figured I'll raise the question? Maybe best to just leave it to the end of the profile work until we remove it anwyay, just checking as I was a bit surprised by this.

Reply:
I don't have the context of whether or not this is still needed but it seems like a good idea to remove it at the end of profile work.